ceph/rbd/workflows/rbd_python.py

```diff
@@ -0,0 +1,60 @@
+import sys
+
+import rados
+import rbd
+from docopt import docopt
+
+doc = """
+A simple test suite wrapper that executes tests based on yaml test configuration
+
+ Usage:
+  rbd_python.py --pool <name>
+        (--image <name>)
+        (--image-size <size>)
+        [--conf-file FILE]
+        [--data-pool <name>]
+
+Options:
+  -h --help                         show this screen
+  --pool <pool>                     pool name
+  --image <image>                   image name
+  --image-size <size>               image size
+  --conf-file <file>                conf file path
+  --data-pool <name>                data pool for ec pool
+"""
+
+
+def rbd_python(args):
+    """
+    Create image and write random data to it, read data written and verify
+    """
+    conf_file = args.get("--conf-file", "/etc/ceph/ceph.conf")
+    pool = args.get("--pool")
+    image_name = args.get("--image")
+    image_size = int(args.get("--image-size"))
+    data_pool = args.get("--data-pool")
+
+    with rados.Rados(conffile=conf_file) as cluster:
+        with cluster.open_ioctx(pool) as ioctx:
+            rbd_inst = rbd.RBD()
+            if data_pool:
+                rbd_inst.create(ioctx, image_name, image_size, data_pool=data_pool)
+            else:
+                rbd_inst.create(ioctx, image_name, image_size)
+            with rbd.Image(ioctx, image_name) as image:
+                data = str.encode("bar" * 20000)
+                image.write(data, 0)
+                mydata = image.read(0, 60000)
+                if data != mydata:
```

Comment:
not cleared about data validation why `60000` for read? Does it bar 3*20000?

Reply:
bar is 3 letters and that multipied by 20000 will actually be 60000 letters written that's why we are reading 60000 back to compare